File: tests/fixtures/structured/java/Exactness.java
package com.example;

import java.util.List;
import java.util.*;

public class Exactness {
    public record Point(int x, int y) {}

    public interface Drawable {
        void draw();
    }

    public enum Color {
        RED, GREEN
    }

    private int field;

    public Exactness() {}

    public void method() {}
}

File: tests/fixtures/structured/csharp/Exactness.cs
using System;
using static System.Math;

namespace MyApp
{
    public class Exactness
    {
        public struct Point
        {
            public int X { get; set; }
        }

        public interface IDrawable
        {
            void Draw();
        }

        public enum Color { Red, Green }

        public record Person(string Name);

        public Exactness() {}

        public void Method() {}
    }
}
